### PR TITLE
Added MV3 and Altered CSS to fix miss-alignment

### PR DESCRIPTION
**Pull request by mort666**
Sorted

### CODE REVIEW

File: build/wide-github.user.css [hunk 1]
```diff
@@ -1,89 +1,90 @@
...
-  }
-
-  /* Refined GitHub extension conflict fix */
-  .refined-github > body:not(.wgh-disabled, .full-width, .enterprise) main > .hide-full-screen > * {
-    padding-left: 0px !important;
-  }
-
-}
+/* ==UserStyle==
+@name         Wide GitHub (RTLabs fork)
```
Comment (Martii) [inline build/wide-github.user.css:2]: :-1:

File: build/wide-github.user.js [hunk 2]
```diff
@@ -1,100 +1,101 @@
...
-(function () {
-  var s = document.createElement('style');
-  s.type = "text/css";
-  s.innerHTML = styleSheet;
-  (document.head || document.documentElement).appendChild(s);
-})();
+"use strict";
+
+// ==UserScript==
+// @name        Wide GitHub (RTLabs fork)
```
Comment (Martii) [inline build/wide-github.user.js:4]: :-1:

File: makejs.js [hunk 3]
```diff
@@ -1,60 +1,60 @@
...
+"// @contributor Paul \"Joey\" Clark (https://github.com/joeytwiddle)\n" +
+"// @contributor Robert Laverty (https://github.com/roblav96)\n" +
+"// @contributor Amir Meimari (https://github.com/amirmeimari)\n" +
+"// @license     MIT; https://raw.githubusercontent.com/xthexder/wide-github/master/LICENSE\n" +
+"// @version     " + manifest["version"] + "\n" +
+"// @icon        https://raw.githubusercontent.com/xthexder/wide-github/master/icons/icon.png\n" +
+"// @homepageURL https://github.com/xthexder/wide-github\n" +
+"// @supportURL  https://github.com/xthexder/wide-github/issues\n" +
+"// @match 			 *://github.com/*\n" +
+"// @grant       none\n" +
```
Comment (Martii) [inline makejs.js:27]: Not exactly sure where else on GH this script/css can apply but this narrows it down significantly i.e. no chance if there is a subdomain.

File: manifest.json [hunk 4]
```diff
@@ -1,27 +1,28 @@
...
-    }
-  ],
-  "background": {
-    "scripts": ["background.js"]
-  }
-}
+{
+  "manifest_version": 3,
+
+  "name": "Wide GitHub (RTLabs fork)",
```
Comment (Martii) [inline manifest.json:4]: :-1:

File: build/wide-github.user.css [hunk 5]
```diff
@@ -81,9 +81,10 @@
     height: calc(100vh - 500px) !important;
   }
 
+  /* Does not appear to be needed anymore - SK 2023-03-29 */
   /* Refined GitHub extension conflict fix */
-  .refined-github > body:not(.wgh-disabled, .full-width, .enterprise) main > .hide-full-screen > * {
+  /* .refined-github > body:not(.wgh-disabled, .full-width, .enterprise) main > .hide-full-screen > * {
```
Comment (Martii) [inline build/wide-github.user.css]: Is this what you are "sorting"? i.e. what issue does this pertain to?

File: build/wide-github.user.js [hunk 6]
```diff
@@ -85,10 +85,11 @@ var styleSheet = "" +
   "height: calc(100vh - 500px) !important;" +
 "}" +
 
+// Does not appear to be needed anymore - SK 2023-03-29
 // Refined GitHub extension conflict fix
-".refined-github > body:not(.wgh-disabled, .full-width, .enterprise) main > .hide-full-screen > * {" +
+/* .refined-github > body:not(.wgh-disabled, .full-width, .enterprise) main > .hide-full-screen > * {
```
Comment (Martii) [inline build/wide-github.user.js]: Is this what you are "sorting"? i.e. what issue does this pertain to?

File: wide-github-toggle.js [hunk 7]
```diff
@@ -1,16 +1,13 @@
...
-  }
-  else
-  {
+  } else {
     document.body.classList.add('wgh-disabled');
   }
 }
 
-chrome.runtime.onMessage.addListener(function(request, sender, callback) {
+chrome.runtime.onMessage.addListener((request, sender, callback) => {
```
Comment (Martii) [inline wide-github-toggle.js]: While technically "okay" you might want to confirm that this is wanted to have newer syntax.

File: wide-github.css [hunk 8]
```diff
@@ -62,7 +62,8 @@ body:not(.wgh-disabled) .gist-content .commit-create .CodeMirror {
   height: calc(100vh - 500px) !important;
 }
 
+/* Does not appear to be needed anymore - SK 2023-03-29 */
 /* Refined GitHub extension conflict fix */
-.refined-github > body:not(.wgh-disabled, .full-width, .enterprise) main > .hide-full-screen > * {
+/* .refined-github > body:not(.wgh-disabled, .full-width, .enterprise) main > .hide-full-screen > * {
```
Comment (Martii) [inline wide-github.css]: Is this what you are "sorting"? i.e. what issue does this pertain to?

**Review by Martii — COMMENTED** [review]
Some definite breaking changes here all around in .user.css, .user.js and probably the extension.

Not entirely sure committing a build of this is wanted because it definitely makes it less easy to evaluate for everyone. i.e. I would usually let the owner of the project build and version bump when ready to release.

File: makejs.js (hunk 3 above)
Comment (xthexder) [reply makejs.js:27]: Yeah, this breaks gist.github.com
The manifest.json for Chrome / Firefox extensions defines "https://github.com/*", "https://gist.github.com/*", which covers everything.

**Comment by mort666** [comment]
Firstly, appologies wasn't meant to PR here, wasn't paying complete attention when doing the PR which was supposed to go specifically to my fork and it was kind of late last night when I was finishing up. But for some reason GH decided to default it to upstream. The 'Title' you thumbs down @Martii was just done so I could track the script/extension on the machines as I was going to test on today.

The changes for MV3 are basically from the same PR that @xthexder did, and given Mozilla/Firefox have [adopted MV3](https://blog.mozilla.org/addons/2022/05/18/manifest-v3-in-firefox-recap-next-steps/) I figured I for my testing I wanted to have the manifest all sorted to ensure it wasn't a 'compliance issue' with MV3 because my whole reason for looking at it was some strange interactions between the userstyles script and Firefox + Refined Github, Brave and Refined Github and and the extension.. Where basically can not highlight text anywhere on Github atm.. 

The commented out thing, well that is the top left where the logo/repo name sitting and Refined Github made a change recently which fixed the thing that the 'workarround' here was negating, and it was doing my OCD in with the top being completely off centre.

**Pull request closed without merging**